tests/pipelines/test_cli.py

```diff
@@ -124,6 +124,31 @@ def test_delete_cli_incorrect_parameters(pipelines_api_mock, tmpdir):
     assert pipelines_api_mock.delete.call_count == 0
 
 
+@provide_conf
+def test_deploy_spec_updated_with_id_if_pipeline_id_not_in_spec(pipelines_api_mock, tmpdir):  # noqa
```

Comment:
what is noqa?

Reply:
It tells the linter to ignore the line. Removing, no longer needed.